nio_cli/commands/config.py

```diff
@@ -14,22 +14,24 @@ def config_project(name='.'):
     pk_token = input('Enter PK Token (optional): ')
     ws_host = pk_host.replace('pubkeeper', 'websocket')
 
-    pk_host_replace = "s/^PK_HOST:.*/PK_HOST: {}/".format(pk_host)
-    pk_token_replace = "s/^PK_TOKEN:.*/PK_TOKEN: {}/".format(pk_token)
-    ws_host_replace = "s/^WS_HOST:.*/WS_HOST: {}/".format(ws_host)
-
-    if pk_host:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_host_replace, env_location),
-            shell=True)
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(ws_host_replace, env_location),
-            shell=True)
-    if pk_token:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_token_replace, env_location),
-            shell=True)
+    nenv = open(env_location, 'r')
+    tmp = open(env_location + '.tmp', 'w')
+    for line in nenv:
+        if re.search('PK_HOST:', line) and pk_host:
+            tmp.write(re.sub('PK_HOST:.*',
+                'PK_HOST: {}'.format(pk_host), line))
+        elif re.search('WS_HOST:', line) and pk_host:
+            tmp.write(re.sub('WS_HOST:.*',
+                'WS_HOST: {}'.format(ws_host), line))
+        elif re.search('PK_TOKEN:', line) and pk_token:
+            tmp.write(re.sub('PK_TOKEN:.*',
+                'PK_HOST: {}'.format(pk_token), line))
+        else:
+            tmp.write(line)
+    nenv.close()
+    tmp.close()
 
+    os.rename(env_location + '.tmp', env_location)
```

Comment:
On windows the original file will need to be removed/renamed before this operation is possible.